src/main/java/seedu/address/model/candidate/predicate/CandidateContainsKeywordsPredicate.java

```diff
@@ -10,7 +10,6 @@
  * Tests that a {@code Candidate} matches any of the keywords given.
  */
 public class CandidateContainsKeywordsPredicate extends ContainsKeywordsPredicate implements Predicate<Candidate> {
-    private final List<String> keywords;
```

Comment:
Oops @charmainehly, I might have to fix this for the `availability` to follow yours. Will KIV it on my end and discuss if needed.